tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AgentMalfunctionTests.cs

```diff
@@ -82,7 +82,13 @@ public async Task SubmitsTraces(AgentBehaviour behaviour, TestTransports transpo
 
         private async Task TestInstrumentation(MockTracerAgent agent, string metadataSchemaVersion)
         {
-            const int expectedSpanCount = 5;
+            int expectedSpanCount = 5;
+            // 3 on non-windows because of SecureString
+            if (!EnvironmentTools.IsWindows())
+            {
+                expectedSpanCount = 3;
+            }
```

Comment:
_tiny_ nit, but I think we tend to use ternary for these
```suggestion
            // 3 on non-windows because of SecureString
            var expectedSpanCount = EnvironmentTools.IsWindows() ? 5 : 3;
```

Reply:
Nice will swap to that